ISSUE_TEMPLATE.md

```diff
@@ -1,17 +1,61 @@
+# Issue Template
+
+Please use this Template to open Issues in this Repository.
+
+```markdown
 Steps to reproduce
 ==================
+
 1. Do this
 2. Do that
 3. Do here
 
+
 Issues
 ======
 * While step 2: that and that is happening
 
+
 Expected
 ========
 * While step 2: that and that should happen
 
+
+Additional Notes
```

Comment:
This is all duplicated now.

Should be kept as short as possible with as much information as possible, for easy editing upon making a new issue.